src/components-examples/angular/tooltip/tooltip-simple/tooltip-simple-example.html

```diff
@@ -0,0 +1,16 @@
+<sbb-tooltip>{{ tooltipContent }}</sbb-tooltip>
+<h4>Properties</h4>
+<sbb-form-field label="Tooltip text content">
+  <input type="text" [(ngModel)]="tooltipContent" sbbInput />
+</sbb-form-field>
+<div>
```

Comment:
I'd suggest to use `<p>` here so the 4k and 5k showcase looks better (scaled font size)